cmd/utils/flags.go

```diff
@@ -1957,14 +1958,17 @@ func MakeGenesis(ctx *cli.Context) *core.Genesis {
 
 // MakeChain creates a chain manager from set command line flags.
 func MakeChain(ctx *cli.Context, stack *node.Node) (chain *core.BlockChain, chainDb ethdb.Database) {
+	return MakeChainWithEngine(ctx, stack, mockEngine.NewFaker())
+}
+
+func MakeChainWithEngine(ctx *cli.Context, stack *node.Node, engine consensus.Engine) (chain *core.BlockChain, chainDb ethdb.Database) {
```

Comment:
It looks like this could be inlined, it seems to only be called from MakeChain

Reply:
seems like a remainder from a previous attempt. removing it